OpenRA.Mods.Cnc/Traits/Render/WithBuildingBib.cs

```diff
@@ -12,18 +12,21 @@
 using System.Collections.Generic;
 using OpenRA.Graphics;
 using OpenRA.Mods.Common.Graphics;
+using OpenRA.Mods.Common.Traits;
 using OpenRA.Mods.Common.Traits.Render;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.Common.Traits
+namespace OpenRA.Mods.Cnc.Traits
 {
-	public class BibInfo : ITraitInfo, Requires<BuildingInfo>, IRenderActorPreviewSpritesInfo, IActorPreviewInitInfo, Requires<RenderSpritesInfo>
+	public class WithBuildingBibInfo : ITraitInfo, Requires<BuildingInfo>, IRenderActorPreviewSpritesInfo, IActorPreviewInitInfo, Requires<RenderSpritesInfo>
 	{
 		[SequenceReference] public readonly string Sequence = "bib";
+
 		[PaletteReference] public readonly string Palette = TileSet.TerrainPaletteInternalName;
-		public readonly bool HasMinibib = false;
 
-		public object Create(ActorInitializer init) { return new Bib(init.Self, this); }
+		public readonly bool IsMiniBib = false;
```

Comment:
A simpler fix would be to remove this flag, forcing the `false` behavior for all uses of this trait, and then change the mini bib actors to use a normal idle overlay.

A future PR can then clean up the positioning and turn this into a general trait for rendering tiled artwork (e.g. TD/RA bridges, d2 buildings).